doc-content/enterprise-only/weblogic/wls-data-source-create-proc.adoc

```diff
@@ -7,6 +7,21 @@ A data source is an object that enables a Java Database Connectivity (JDBC) clie
 * The JDBC drivers that you want to use to create database connections are installed on all servers on which you want to deploy the data source. Some JDBC drivers are installed with {WEBLOGIC}, such as WebLogic-branded Data Direct JDBC drivers for DB2, Informix, MS SQL Server, and Sybase. For more information about JDBC drivers, see https://docs.oracle.com/middleware/12213/wls/JDBCA/third_party_drivers.htm#JDBCA231[Using JDBC Drivers with WebLogic Server] in the Oracle Help Center.
 
 .Procedure
+. Navigate to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required), and select the product and version from the drop-down options:
+* *Product:* {PRODUCT_SHORT}
+* *Version:* {PRODUCT_VERSION}
+. Download *{PRODUCT} {PRODUCT_VERSION}.x Add-Ons*, where *x* is the current patch version.
+. Prepare your database:
+.. Unzip `{PRODUCT_INIT}-{PRODUCT_VERSION}.x-add-ons.zip` in a temporary directory, for example `TEMP_DIR`.
+.. Unzip `TEMP_DIR/{PRODUCT_INIT}-{PRODUCT_VERSION}-migration-tool.zip`.
+.. Change directory to the `TEMP_DIR/{PRODUCT_INIT}-{PRODUCT_VERSION}-migration-tool/ddl-scripts` directory. This directory contains DDL scripts for several database types.
```

Comment:
It is really unclear what the user is expected to do. Are you just asking them to go to a different directory to fetch the scripts?